seed/challenges/01-front-end-development-certification/html5-and-css.json

```diff
@@ -1576,7 +1576,7 @@
       ],
       "tests": [
         "assert((/cat photos/gi).test($(\"a\").text()), 'message: Your <code>a</code> element should have the <code>anchor text</code> of \"cat photos\".');",
-        "assert(/http:\\/\\/freecatphotoapp\\.com/gi.test($(\"a\").attr(\"href\")), 'message: You need an <code>a</code> element that links to <code>http&#58;//freecatphotoapp<wbr>.com</code>');",
+        "assert((/http:\\/\\/freecatphotoapp\\.com/gi.test($(\"a\").attr(\"href\")) || /http:\\/\\/www.freecatphotoapp\\.com/gi.test($(\"a\").attr(\"href\"))), 'message: You need an <code>a</code> element that links to <code>http&#58;//freecatphotoapp<wbr>.com</code>');",
```

Comment:
I think this could be shortened with the use of a capturing group

```js
/(this|that)\ssenctence/gi
```
This regEx will match `'this sentence'` or `'that senctence'`.